integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/stack/StackTestDtoBase.java

```diff
@@ -263,6 +263,11 @@ private InstanceGroupV4Request getInstanceGroupV4Request(String key) {
         return this;
     }
 
+    public StackTestDtoBase<T> withNetworkOfEnvironment() {
```

Comment:
Can we have only one method for setting the network to null?
withEmtyNetwork does exactly the same